daffodil-core/src/main/scala/org/apache/daffodil/dpath/DFDLExpressionParser.scala

```diff
@@ -270,34 +276,41 @@ class DFDLPathExpressionParser[T <: AnyRef](
       RelativePathExpr)("path")
 
   def RelativePathExpr: Parser[RelativePathExpression] = log(
-    StepExpr ~ ("/" ~> StepExpr).* ^^ { case s1 ~ moreSteps => RelativePathExpression(s1 :: moreSteps, isEvaluatedAbove) } |
-      StepExpr ~ ("//" ~> StepExpr).* ^^ { _ => context.SDE("'//' is unsupported in DFDL Expression Syntax.") })("relativePath")
+    StepExpr ~ ("/" ~> StepExpr).* ^^ { case s1 ~ moreSteps =>
+      RelativePathExpression(s1 :: moreSteps, isEvaluatedAbove)
+    } |
+      StepExpr ~ ("//" ~> StepExpr).* ^^ { _ => context.SDE("'//' is unsupported in DFDL Expression Syntax.") })(
+    "relativePath")
 
-  def StepExpr: Parser[StepExpression] = log(AxisStep | VarRef ^^ { varRef => this.context.SDE("Variables cannot be used in path expressions.  Error: $%s", varRef.qnameString) })("step")
+  def StepExpr: Parser[StepExpression] = log(AxisStep | VarRef ^^ { varRef =>
+    this.context.SDE("Variables cannot be used in path expressions.  Error: $%s", varRef.qnameString)
+  })("step")
 
   def AxisStep: Parser[StepExpression] =
     Reverse | Forward
 
   def Reverse = AbbrevReverseStep ~> Predicate.? ^^ { Up(_) } |
     (SupportedReverseAxis ~> NodeTest) ~ Predicate.? ^^ { case qn ~ p => { Up2(qn, p) } } |
-    (UnsupportedReverseAxis ~ NodeTest) ~ Predicate.? ^^ { case name ~ _ => context.SDE("'%s::' is an unsupported axis in DFDL Expression Syntax.", name) }
+    (UnsupportedReverseAxis ~ NodeTest) ~ Predicate.? ^^ { case name ~ _ =>
+      context.SDE("'%s::' is an unsupported axis in DFDL Expression Syntax.", name)
+    }
 
   def Forward = ContextItemExpr ~> Predicate.? ^^ { Self(_) } |
     SupportedForwardAxis ~ NodeTest ~ Predicate.? ^^ {
-      case "self" ~ qn ~ p => Self2(qn, p)
+      case "self" ~ qn ~ p  => Self2(qn, p)
       case "child" ~ qn ~ p => NamedStep(qn, p)
       // $COVERAGE-OFF$
       case _ => Assert.impossible()
       // $COVERAGE-ON$
     } |
-    UnsupportedForwardAxis ~ Predicate.? ^^ { case name ~ _ => context.SDE("'%s::' is an unsupported axis in DFDL Expression Syntax.", name) } |
+    UnsupportedForwardAxis ~ Predicate.? ^^ { case name ~ _ =>
+      context.SDE("'%s::' is an unsupported axis in DFDL Expression Syntax.", name)
+    } |
     NodeTest ~ Predicate.? ^^ { case qn ~ p => { NamedStep(qn, p) } }
 
-  def Predicate: Parser[PredicateExpression] = log(
-    "[" ~> Expr <~ "]" ^^ { PredicateExpression(_) })("predicate")
+  def Predicate: Parser[PredicateExpression] = log("[" ~> Expr <~ "]" ^^ { PredicateExpression(_) })("predicate")
```

Comment:
The line length being used here is much too long.  (113 chars).

I think the max should be 96 chars / line.

Reply:
> Lines are far too long.

The default was 80 and it was reformatting a lot because there are quite a few long lines in the code. I bumped it up to 120 but that is quite long. I've moved it down to 96. See how that looks. 

> Lining up the arrows in match-case clauses is simply annoying and should be turned off.

Alignment has been disabled. 

> Changes are also needed to the way extends and with (one line per) for traits/classes are formatted.

What specifically would you like changed?

Reply:
I withdraw the request about extends and with.

I took a look at the ElementBase trait and the indentation was not changed:

```
trait ElementBase
  extends Term
  with ElementLikeMixin
  with LocalElementMixin
  with Element_AnnotationMixin
  with NillableMixin
  with ElementBaseGrammarMixin
  with ElementRuntimeValuedPropertiesMixin
  with StringTextMixin
  with NumberTextMixin
  with CalendarTextMixin
  with BooleanTextMixin
  with TextNumberFormatMixin
  with EmptyElementParsePolicyMixin
  with TextStandardBaseMixin
  with OverlapCheckMixin
  with ElementBaseView {
```

So I'm fine with that.